### DISCUSSION TITLE
v0.6.4

### DISCUSSION THREAD

**Discussion by yoavrotems** [aquasecurity/kube-bench#987] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/aquasecurity/kube-bench/releases/tag/v0.6.4'>v0.6.4</a>.</em>

**Comment by marcinbojko** [comment]
Missing packages for 0.6.4

**Reply by yoavrotems** [reply]
Fixed in v0.6.5 thanks for reporting.
